src/nettests/ndt/index.js

```diff
@@ -0,0 +1,101 @@
+exports.formatRunSummary = (summary, {React, Cli, Components, chalk}) => {
+  const uploadMbit = Util.toMbit(summary.upload)
+  const downloadMbit = Util.toMbit(summary.download)
+  const ping = Math.round(summary.ping*10)/10
+  const packetLoss = Math.round(summary.packetLoss * 100 * 100)/100
+  const outOfOrder = Math.round(summary.outOfOrder * 100 * 100)/100
+
+  const mss = summary.mss
+  const timeouts = summary.timeouts
+
+  // When this function is called from the Cli the Cli will be set, when it's
+  // called from the Desktop app we have React set instead.
+  if (Cli) {
```

Comment:
I was wondering whether perhaps having a common interface (be it a callable or an object) that wraps the `Cli` and `React` and hides their differences could be something interesting for us here.